nexus/analyzer/src/lib.rs

```diff
@@ -803,6 +803,37 @@ fn parse_db_options(
             let config = Config::EventhubGroupConfig(eventhub_group_config);
             Some(config)
         }
+        DbType::Clickhouse => {
+            let s3_int = opts
```

Comment:
is there a reason we went with an empty string being the default instead of `optional` like with metadata_schema?

Reply:
I have replicated from the snowflake config. As there would be common code dealing with s3 config it is easier to keep it same as that of snowflake.

Reply:
makes sense